py/htm/examples/mnist.py

```diff
@@ -67,32 +66,36 @@ def load_images(file_name):
             assert(len(raw) == data_start + img_size * num_imgs)   # All data should be used.
         return imgs
 
-    train_labels = load_labels(os.path.join(path, 'train-labels-idx1-ubyte.gz'))
-    train_images = load_images(os.path.join(path, 'train-images-idx3-ubyte.gz'))
-    test_labels  = load_labels(os.path.join(path, 't10k-labels-idx1-ubyte.gz'))
-    test_images  = load_images(os.path.join(path, 't10k-images-idx3-ubyte.gz'))
+    train_labels = load_labels(os.path.join(path, 'train-labels-idx1-ubyte'))
+    train_images = load_images(os.path.join(path, 'train-images-idx3-ubyte'))
+    test_labels  = load_labels(os.path.join(path, 't10k-labels-idx1-ubyte'))
+    test_images  = load_images(os.path.join(path, 't10k-images-idx3-ubyte'))
 
     return train_labels, train_images, test_labels, test_images
 
-
+# these parameters can be improved using parameter optimization, 
+# see py/htm/optimization/ae.py
```

Comment:
I did run parameter optimization, that's how I found the original parameters which we're now changing.

Reply:
it's more of a note that it's possible to optimize that here. Still, if you change the hyperparams of the optimization (eg the starting point, runtime, ...) you end up with more or less different values.